oneview/resource_logical_enclosure.go

```diff
@@ -171,26 +173,28 @@ func resourceLogicalEnclosure() *schema.Resource {
 			"power_mode": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
 			},
 			"scaling_state": {
 				Type:     schema.TypeString,
-				Optional: true,
+				Computed: true,
 			},
 			"scopes_uri": {
 				Type:     schema.TypeString,
 				Optional: true,
```

Comment:
can we change scopes_uri once element created?

Reply:
No scope uri can not be changed